packages/contracts-bedrock/src/dispute/interfaces/IFaultDisputeGame.sol

```diff
@@ -63,6 +71,8 @@ interface IFaultDisputeGame is IDisputeGame {
 
     /// @notice The l2BlockNumber that the `rootClaim` commits to. The trace being bisected within
     ///         the game is from `l2BlockNumber - 1` -> `l2BlockNumber`.
-    /// @return l2BlockNumber_ The l2BlockNumber that the `rootClaim` commits to.
     function l2BlockNumber() external view returns (uint256 l2BlockNumber_);
+
+    /// @notice The l1BlockNumber that Cannon was ran from to generate the root claim.
+    function l1BlockNumber() external view returns (uint256 l1BlockNumber_);
```

Comment:
We should be natspeccing where we can including return values to be consistent with our own style guides, but not a big deal

Reply:
In a few other PR's, you nitted that the `/// @return` for these self explanatory getters was unnecessary - do we want to put something in the style guide for this?

Reply:
Oops, totally my fault. Yes, in a linear ticket, let's decide on whether to document in a linear ticket and then add to the style guide.

Created ticket in gardening: https://linear.app/optimism/issue/CLI-4323/docs-decide-and-add-to-style-guide-when-returns-are-worth-documenting